### DISCUSSION TITLE
Not able to get the Setup wizard *solved*

### DISCUSSION THREAD

**Discussion by Zulu-Zap** [cryinkfly/Autodesk-Fusion-360-for-Linux#219] [Q&A] [answered]
When I use the following command it seems I am not able to download the setup wizard. Has anything changed or did I make a mistake?

$ wget -N https://raw.githubusercontent.com/cryinkfly/Autodesk-Fusion-360-for-Linux/main/files/scripts/stable-branch/fusion360-setup-wizard.sh

![image](https://user-images.githubusercontent.com/54446766/165055344-f25351ab-daea-4719-b0fe-0f9275e746fa.png)

**Comment by Zulu-Zap** [comment, marked as answer]
I am running Linux Mint currently

**Reply by cryinkfly** [reply]
The link is broken! Please use this command:

    mkdir -p "$HOME/.config/fusion-360/bin" && cd "$HOME/.config/fusion-360/bin" && wget -N https://raw.githubusercontent.com/cryinkfly/Autodesk-Fusion-360-for-Linux/main/files/builds/stable-branch/bin/install.sh && chmod +x install.sh && bash install.sh && exit
